src/107-Arduino-Servo-RP2040.h

```diff
@@ -32,23 +32,29 @@ class Servo
   static uint16_t constexpr DEFAULT_MIN_PULSE_WIDTH_us     = 1000;
   static uint16_t constexpr DEFAULT_MAX_PULSE_WIDTH_us     = 2000;
   static uint16_t constexpr DEFAULT_NEUTRAL_PULSE_WIDTH_us = 1500;
+  static uint16_t constexpr DEFAULT_MAX_ANGLE              = 180;
 
   static uint16_t constexpr PWM_PERIOD_us = 20*1000UL;
 
   Servo()
   : _min_pulse_width_us{DEFAULT_MIN_PULSE_WIDTH_us}
   , _max_pulse_width_us{DEFAULT_MAX_PULSE_WIDTH_us}
   , _neutral_pulse_width_us{DEFAULT_NEUTRAL_PULSE_WIDTH_us}
+  , _max_angle{DEFAULT_MAX_ANGLE}
   , _is_attached{false}
   , _slice_num{0}
   , _channel{0}
   { }
 
   void attach(pin_size_t const pin);
+  void attach(pin_size_t const pin, uint16_t min_pulse_width_us, uint16_t max_pulse_width_us);
+  void setMaxAngle(uint16_t const max_angle);
```

Comment:
`write` and `writeAngle` take a `int` argument, `setMaxAngle` an `uint16_t`: I'd reason that it should be a `int` too. What do you think?

Reply:
I had overlooked it, I agree. Thanks for the review.